### PR TITLE
API Final Project

### PR DESCRIPTION
**Pull request by Simcha2023**
Hi Laura, sorry for the delay, here it is.

### CODE REVIEW

File: student-work/2023/Sergey-Boroditsky/api-final-project/get.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# GET Documentation
+
+### Description:
+Utilize the GPMD APIs to interact with the program's functionalities, such as placing orders and requesting bills. The GET method is utilized for initiating a bill request, while the POST method is used for creating a meal request. For further details about this API, refer to the README file.
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sergey-Boroditsky/api-final-project/get.md:4]: the word README should be a link to the readme file 
for example
[README](readme.md)

File: student-work/2023/Sergey-Boroditsky/api-final-project/get.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+### HTTP Method:
+GET
+### Request Parameters:
+- tableNo: the number assigned to the table. For takeaway orders, the table number is 99.
+- password: password for authentification
+### Request Scheme:
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sergey-Boroditsky/api-final-project/get.md:10]:
```suggestion
### Request Schema:
```

File: student-work/2023/Sergey-Boroditsky/api-final-project/get.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+```
+curl -X GET "https://meal/burgerMeal?tableNo=5&password=your_password"
+```
+### Response Codes:
+| **Code** | **Status** | **Description** |
+|---|---|---|
+| 200 | OK | Request successfully processed |
+| 315 | Wrong parameter | Invalid table number. Check tableNo. |
+| 400 | Unauthorized | Invalid credentials. Check password. |
+| 535 | General error | Something else went wrong. Generic error. |
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sergey-Boroditsky/api-final-project/get.md:25]: What should you do if this error happens?

File: student-work/2023/Sergey-Boroditsky/api-final-project/get.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+### Response Scheme:
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Sergey-Boroditsky/api-final-project/get.md:27]:
```suggestion
### Response Schema:
```

**Review by Laura-Novich-OBW — COMMENTED** [review]
Please make the changes requested

**Comment by Laura-Novich-OBW** [comment]
otherwise it is looking good.

**Comment by Simcha2023** [comment]
Hi Laura, I made the necessary corrections.

**Review by Laura-Novich-OBW — APPROVED** [review]

**Comment by Laura-Novich-OBW** [comment]
nice job!

**Pull request merged**